cookbooks/aws-parallelcluster-platform/resources/install_packages/install_packages_redhat8.rb

```diff
@@ -23,12 +23,13 @@ def default_packages
   # environment-modules required by EFA, Intel MPI and ARM PL
   # Removed libssh2-devel from base_packages since is not shipped by RedHat 8 and in conflict with package libssh-0.9.6-3.el8.x86_64
   # iptables needed for IMDS setup
+  # ToDo send a message in pcluster-interest to see if the package removal/addition is ok
```

Comment:
Let's remove this comment about pcluster interest

Reply:
Done